### DISCUSSION TITLE
[Script request] Lubelog

### DISCUSSION THREAD

**Discussion by Mightyretro** [community-scripts/ProxmoxVE#478] [Request script]
### Application Name

Lubelog

### Website

https://github.com/hargata/lubelog

### Description

LubeLogger is a web-based vehicle maintenance and fuel mileage tracker

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Duplicate of: https://github.com/community-scripts/ProxmoxVE/discussions/48
